tests/extended/test_trainer_ext.py

```diff
@@ -202,6 +202,7 @@ def test_run_seq2seq(self):
         assert "predict_results.json" in contents
 
     @slow
+    @require_torch_multi_gpu
```

Comment:
So far I have no idea why we don't observe reduced GPU memory usage on single GPU env.